packages/hub/src/lib/commit.spec.ts

```diff
@@ -33,24 +32,24 @@ describe("commit", () => {
 		try {
 			await commit({
 				repo,
-				title:       "Some commit",
+				title: "Some commit",
 				credentials: {
 					accessToken: TEST_ACCESS_TOKEN,
 				},
 				operations: [
 					{
 						operation: "addOrUpdate",
-						content:   new Blob(["This is me"]),
-						path:      "test.txt",
+						content: new Blob(["This is me"]),
+						path: "test.txt",
 					},
 					{
 						operation: "addOrUpdate",
-						content:   new Blob([lfsContent]),
-						path:      "test.lfs.txt",
+						content: new Blob([lfsContent]),
+						path: "test.lfs.txt",
 					},
 					{
 						operation: "delete",
-						path:      "README.md",
+						path: "README.md",
```

Comment:
Can you run `pnpm -r format`?

I'll add a lint action

Reply:
Not sure why my editor isn't picking up the config. Does the repo use eslint for formatting?

Reply:
Uses a custom extension (recommended)